src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -732,7 +833,8 @@ void WindowImplWin32::processEvent(UINT message, WPARAM wParam, LPARAM lParam)
                 event.key.control = HIWORD(GetAsyncKeyState(VK_CONTROL)) != 0;
                 event.key.shift   = HIWORD(GetAsyncKeyState(VK_SHIFT))   != 0;
                 event.key.system  = HIWORD(GetAsyncKeyState(VK_LWIN)) || HIWORD(GetAsyncKeyState(VK_RWIN));
-                event.key.code    = virtualKeyCodeToSF(wParam, lParam);
+                event.key.scancode = toScancode(lParam);
+                event.key.code = virtualKeyCodeToSF(wParam, lParam);
```

Comment:
I'm a bit confused by what's happening here. Can't we map scancode that came from windows event to virtual key and then only use it for scancode implementation? I think it's better to map virtual key codes to sf::Keyboard::Scancode than take specific scancodes into account (those are hardware specific, AFAIK)